src/wymeditor/parser/xhtml-sax-listener.js

```diff
@@ -415,7 +417,17 @@ WYMeditor.XhtmlSaxListener.prototype.closeBlockTag = function(tag) {
         this._insideLI = false;
     }
 
-    this.output = this.output.replace(/<br \/>$/, '') +
+    // If a `br` is the only tag within the element that is being closed,
+    // remove that `br` tag.
+    this.output = this.output.replace(
+        new RegExp(
+            this._lastAddedOpeningTagString + '<br />$',
```

Comment:
If this has characters that need to be escaped in a regex, we can get a SyntaxError.